torch/ao/quantization/pt2e/graph_utils.py

```diff
@@ -22,7 +22,7 @@
     {torch.nn.ReLU, torch.nn.functional.relu, torch.nn.functional.relu_},
     {torch.nn.BatchNorm2d, torch.nn.functional.batch_norm},
     {torch.nn.Hardtanh, torch.nn.functional.hardtanh, torch.nn.functional.hardtanh_},
-    {torch.add, operator.add, operator.iadd},
+    {torch.add, operator.add, operator.iadd, "add", "add_"},
```

Comment:
Hi @kimishpatel, I think we need to extend the `_EQUIVALENT_TYPES` list with "add", "add_" for cases of `x.add(y)` and `x.add_(y)`. Please kindly help to take a look.

Reply:
why is add_ here? Exported program should be functional and thus should not have in place variants like add_

Reply:
I think here we should configure the `source_fn` of the node: https://github.com/pytorch/pytorch/blob/93e6fc54fac359d0fd5e314847375086c01cedf2/torch/fx/passes/utils/source_matcher_utils.py#L78. So the `source_fn` may include the in place variants like `operator.iadd` which is already in the list.

Reply:
are these needed? and are these actually taking effect?

Reply:
yean, I think it's needed for users who writes `x.add(y)` or `x.add_(y)`.